ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+Please try to adhere to the guidelines below as much as possible when
+submitting your issue.
+- Verify that your issue is not being currently addressed by other
+[issues](https://github.com/scikit-learn/scikit-learn/issues?q=)
+or [pull requests](https://github.com/scikit-learn/scikit-learn/pulls?q=).
+- If your issue is a usage question or does not potentially require
+changes to the codebase to be solved, then StackOverflow (using the
+`[scikit-learn]` tag) or our [mailing list](https://lists.sourceforge.net/lists/listinfo/scikit-learn-general)
+may be a better place to bring it up. For more information, see
+["User Questions"](https://lists.sourceforge.net/lists/listinfo/scikit-learn-general).
```

Comment:
No need for the double quotes (there are other places where you do that, please fix it).